### DISCUSSION TITLE
Płynne przewijanie strony, do wskazanego punktu

### DISCUSSION THREAD

**Discussion by giscus** [robert-orlinski/robertorlinski.pl#18] [Komentarze]
# Płynne przewijanie strony, do wskazanego punktu

W artykule piszę o trzech sposobach, które pozwolą nam dodać płynne przewijanie do wskazanego elementu na swojej stronie internetowej 🌊

https://robertorlinski.pl/artykuly/plynne-przewijanie-witryny

**Comment by grabek-77** [comment]
A jak spowodować automatyczne przewinięcie strony (nie musi być płynne) do samej góry po jej załadowaniu ?
Wiem, że może wydawać się bez sensu, ale potrzebuje takiego rozwiązania, ponieważ anchorami rozwijam element accordion, i strona pozostaje na wysokości danego elementu accordion zamiast pokazać się od samej góry. Podpowiesz ? :)

**Reply by robert-orlinski** [reply]
Hej, `document.body.scrollIntoView()` z czystego JSa, podpięte pod otwarcie accordionu powinno wystarczyć :) Jeśli chcesz przescrollować stronę płynnie, to możesz też użyć `document.body.scrollIntoView({ behavior: "smooth" })`.

Więcej: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView.
